app/controllers/forms/live_controller.rb

```diff
@@ -2,20 +2,25 @@ class Forms::LiveController < Forms::BaseController
   after_action :verify_authorized
   def show_form
     authorize current_form, :can_view_form?
-    render template: "live/show_form", locals: { form_metadata: current_form, form: current_live_form, metrics_data: }
+    render template: "live/show_form", locals: { form_metadata: current_form, form: current_live_form, form_live_date:, metrics_data: }
   end
 
   def show_pages
     authorize current_form, :can_view_form?
     render template: "live/show_pages", locals: { form: current_live_form }
   end
 
+  def form_live_date
+    Time.zone.parse(current_live_form.live_at.to_s).to_date
+  end
```

Comment:
This doesn't feel like the right place for this method.

1. This is essentially a decorated value we would like for the view so should be in model or somewhere other than a controller.
2. Are we going to use this anywhere else at the moment? if not maybe we could just inline it in the view if theres no clear place to put this.

Reply:
Fair point - I feel like putting it in the `form` model would probably make most sense. 

Part of me wonders if it would make sense to do the same with `metrics_data`, and then just pass `form` into the component (but that's a bigger change that I'd want to do in a separate PR, if I do it)

Reply:
Have moved it onto the form model - think it's much better now so thanks!